cax/main.py

```diff
@@ -288,10 +292,48 @@ def massive():
         docs = list(collection.find(query,
                                     sort=sort_key,
                                     projection=['start', 'number','name',
-                                                'detector', '_id']))
+                                                'detector', '_id', 'processor']))
 
         for doc in docs:
+            
+            #If no_batch_mode is chosen, no jobs will submitted to batch queues
+            if no_batch_mode == True:
+                
+                #Define basic command:
+                basic_command = """#!/bin/bash
+cax --once {config} --name {name}
+"""
+                
+                print("no-batch-mode")
+                
+                #Pre-read the config file for the no-batch-mode modus:
+                config.set_json(os.path.abspath(args.config_file))
+                task_list = config.get_config(config.get_hostname())['task_list']
+                
+                #This is a list of tasks which refer to apply corrections to a run:
+                task_list_correction = ['AddElectronLifetime', 'AddGains', 'AddDriftVelocity', 'SetS2xyMap', 'SetLightCollectionEfficiency', 'SetFieldDistortion', 'SetNeuralNetwork']
+                
+                #--------------------
+                #Start with single activities: Apply corrections only to data sets which do not have any correction yet:
+                if bool(set(task_list) & set(task_list_correction)) or task_list_correction == task_list:
```

Comment:
Yes it is included and you would not mind if you run the full set of corrections such as they are defined in line (https://github.com/XENON1T/cax/blob/144fd4e651266e85adafecc01e845572eafbe1ca/cax/main.py#L314). But in case you run only a subset of corrections it will need to math the condition before the "or".

In case we come up with more corrections we would need to adjust line 314. But I think this is ok.